includes/class-wc-stripe-api.php

```diff
@@ -242,6 +263,13 @@ public static function retrieve( $api ) {
 			]
 		);
 
+		// If we get a 401 error, we know the secret key is not valid, we save a transient to avoid making calls until the secrect key gets updated.
+		if ( ! empty( $response['response']['code'] ) && 401 === $response['response']['code'] ) {
+			$transient_key = WC_Stripe_Mode::is_test() ? self::TEST_MODE_INVALID_API_KEYS_TRANSIENT_KEY : self::LIVE_MODE_INVALID_API_KEYS_TRANSIENT_KEY;
```

Comment:
We should still have the local variable from the top of the function available. Maybe we can call it `$invalid_key_transient_key` and re-use the value from above?